.changeset/tiny-balloons-count.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/combobox': patch
+---
+
+Ensures the focus remains in the combobox when a chip is removed. Focus is set to either the input, or the next chip to the right, whichever is relevant
```

Comment:
When you say, `the next chip to the right`, do you mean the last chip? I was testing with five chip and when i removed the first chip the last chip became focused.

Reply:
Hmm yeah, I just noticed that too. I'll fix that

Reply:
@shaneeza Should be fixed now!